readiness-tool/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "chrome-ready-ness-tool",
+  "version": "10.3.1",
+  "description": "A tool for the sales team to better market AMP",
+  "author": "Alanna Walton <alwalton@google.com>",
```

Comment:
Usually we don't have author credits since they get out of date quickly. 

Copy amphtml project here: "The AMP HTML Authors"